data/data-pipeline/data_pipeline/etl/score/config/datasets.yml

```diff
@@ -0,0 +1,49 @@
+---
+datasets:
+  - long_name: "FEMA National Risk Index"
+    short_name: "nri"
+    module_name: national_risk_index
+    description_short: "Dataset from FEMA that identifies communities most at risk to 18 natural hazards."
+    description_long: "Dataset from FEMA that identifies communities most at risk to 18 natural hazards."
+    input_geoid_tract_field_name: "TRACTFIPS"
+    load_fields:
+      - short_name: "ex_loss"
```

Comment:
Since this NRI class is the "canonical" refactor example, can we include all of the optional fields and explicitly set them to their default values (for the bools i think the defaults are all `False`)?

Reply:
Will pair with @emma-nechamkin on setting these up.

Reply:
I don't really understand what this means? We have fields that go to percentiles, reverse percentiles, and booleans. Low median income is a reverse percentiles -- anything with "low" is. Booleans are for FUDS, eAMLIS, and HRS data. The other special case here is imputation. It will be hard to do the imputation based on geographic neighbors here.

Reply:
> I don't really understand what this means? We have fields that go to percentiles, reverse percentiles, and booleans. Low median income is a reverse percentiles -- anything with "low" is. Booleans are for FUDS, eAMLIS, and HRS data. The other special case here is imputation. It will be hard to do the imputation based on geographic neighbors here.

I have no idea what this means and that's why I'd be grateful if @emma-nechamkin and @lucasmbrown-usds hash this part out.

Reply:
@lucasmbrown-usds fixed this.